__tests__/app.test.js

```diff
@@ -17,4 +18,24 @@ describe('app routes', () => {
   afterAll(() => {
     return mongoose.connection.close();
   });
+
+  it('gets all recipes', async() => {
```

Comment:
I don't think you need this test since you're already have it in your recipe.test.js file.